lib/puppet-ghostbuster/puppetdb.rb

```diff
@@ -1,36 +1,41 @@
 require 'puppetdb'
 require 'puppet'
-require 'puppet/util/puppetdb'
+# require 'puppet/util/puppetdb'
```

Comment:
I think we should make this optional. something like this:
```
begin
  require 'voxpupuli/rubocop/rake'
puppetdb = Puppet::Util::Puppetdb.config.server_urls[0]}
rescue LoadError
  puppetdb = "https://#{Puppet[:server]}:8081"
end

and then use that in line 11

Reply:
tried to get that to work, but predefining variables outside of a class without using global variables and therefore making rubocop happy is not working.
Is there a reason we should even try to load puppet::util::puppetdb? as far as i am concerned we have a hardcoded defaultvalue which we mention in the readme. The other case is setting the env variable. We cover both of these cases right now. Is there something I am missing?

Reply:
the nice thing about puppet::util::puppetdb is that it reads the local puppetdb.conf. You can move the whole block, including the require statement, into the class.